graphistry/pygraphistry.py

```diff
@@ -316,48 +268,47 @@ def _handle_auth_url(auth_url, sso_timeout, sso_opt_into_type):
                     token = None
             if token:
                 # set org_name to sso org
-                PyGraphistry._config['org_name'] = org_name
+                self.session.org_name = org_name
 
                 print("Successfully logged in")
-                return PyGraphistry.api_token()
+                return self.api_token()
             else:
                 print("Please run graphistry.sso_get_token() to complete the authentication after you have authenticated via SSO")
                 return None
         else:
             # print("Start getting token ...")
             # token = None
             # for i in range(10):
-            #     token, org_name = PyGraphistry._sso_get_token()
+            #     token, org_name = self._sso_get_token()
             #     if token:
             #         # set org_name to sso org
-            #         PyGraphistry._config['org_name'] = org_name
+            #         self._config['org_name'] = org_name
             #         print("Successfully logged in")
-            #         return PyGraphistry.api_token()
+            #         return self.api_token()
             #     print("Keep trying to get token ...")
             #     time.sleep(5)
 
             print("Please run graphistry.sso_get_token() to complete the authentication")
             return None
 
-    @staticmethod
-    def sso_get_token():
+    def sso_get_token(self) -> Optional[str]:
```

Comment:
i think moving these from global static exports to exporting an instance method (see bottom of this file) is technically a breaking change

that was a bad enough design that i don't think it gets used in practice. so maybe the answer is the change is ok, and we get rid of the exports at the bottom of this file as make even less sense now?

Reply:
but then the __init__ still needs to export all the methods?

are you suggesting that people write:
```
from graphistry import PyGraphistry as graphistry
```

probably not?

I am not sure what you mean